### DISCUSSION TITLE
Categorical vs. numeric variables in 5.0

### DISCUSSION THREAD

**Discussion by vemuribv** [now-i-know-my-abcd/docs#70] [Stats & Code Help] [answered]
Is there a recommended way to specify categorical vs. numeric variables in release 5.0? (Context: working with several hundreds of variables at a time for ML analyses in Python)

I found [this](https://github.com/ABCD-STUDY/analysis-nda/blob/master/ABCD%20RDS%204.0/05_categorical_extension_4_0.R) for 4.0 but, as far as I know, the tabulated data structure may have changed with the 5.0 release. 

I know that many (most? all?) categorical variables have their category labels listed in the "notes" column of the data dictionary, so maybe I could mark variables with a non-empty notes column as categorical? Though this strategy isn't ideal

**Comment by angielaird** [comment, marked as answer]
Hi! The two approaches you describe are possible strategies. But unfortunately, no, there is currently no recommended way to specify categorical vs. numeric.
